### ISSUE TITLE
DynamicOverflow - set overflow on start or end

### ISSUE THREAD

**Issue by WinXaito** [bdlukaa/fluent_ui#1134]
**Is your feature request related to a problem? Please describe.**

I used the widget `DynamicOverflow`, and it's great but I would really like to have to widget on the left **and** to have the overflow maybe from the start of the list.

There is what I have:

![image](https://github.com/user-attachments/assets/c8b63a18-ba9c-4625-a33a-a92be2abae1c)

And what I want:

![image](https://github.com/user-attachments/assets/5d993e35-e338-4666-8055-f60019b1cca0)

(I want to keep the end of the path and hide the beggining, a little like Windows Explorer. And having the overflow widget on the left).

**Describe the solution you'd like**
Having a parameter like

```
overflowAlignment: MainAxisAlignment.start
```

(It's different from `overflowWidgetAlignment`).

**Comment by bdlukaa** [comment]
Maybe you want to use the [`BreadcrumbBar` widget](https://bdlukaa.github.io/fluent_ui/#/navigation/breadcrumb_bar)?

**Comment by WinXaito** [comment]
Yes that's what I finally did (sorry, forgot to update the issue).

However, do you think it could be possible to customize the BreadcrumbItem ? (I would like to have an effect like a button).
